statsd/uds.go

```diff
@@ -31,6 +31,18 @@ func newUDSWriter(addr string, writeTimeout time.Duration, transport string) (*u
 	return writer, nil
 }
 
+// GetTransportName returns the transport used by the writer
+func (w *udsWriter) GetTransportName() string {
+	if w.transport == "unixgram" {
```

Comment:
I think `w.transport` access needs a lock to avoid a data race.

Reply:
done, and removed a few string comparisons